__tests__/app.test.js

```diff
@@ -0,0 +1,44 @@
+const request = require('supertest');
+const app = require("../app");
+const seed = require('../db/seeds/seed');
+const db = require('../db/connection');
+const testData = require("../db/data/test-data");
+
+afterAll(() => 
+  db.end()
+);
+
+beforeEach(() => seed(testData));
+
+describe('API',() => {
+  test("404: if no page is found return status 404", () => {
+    return request(app)
+      .get("/asjdfh")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body).toEqual({ msg: "Page Not Found" });
```

Comment:
Don't think about this from a FE point of view, Not Found enough as this is an API